index.js

```diff
@@ -11,14 +11,21 @@ if (other) {
   }
 } else
 if (osx) {
-  var regPath = '/Applications/Google Chrome.app/Contents/MacOS/Google Chrome'
+  var osxSuffix = '/Contents/MacOS/Google Chrome'
+  var regPath = '/Applications/Google Chrome.app' + osxSuffix
   var altPath = require('userhome')(regPath.slice(1))
+  var mdFindCmd = 'mdfind \'kMDItemDisplayName == "Google Chrome" && kMDItemKind == Application\''
 
-  module.exports = fs.existsSync(regPath)
-    ? regPath
-    : altPath
+  if (fs.existsSync(regPath)) {
+    module.exports = regPath
+  } else if (fs.existsSync(altPath)) {
+    module.exports = altPath
+  } else {
+    var foundPath = require('child_process').execSync(mdFindCmd, { encoding: 'utf8' })
```

Comment:
Yeah ... I'm generally kind of confused as to what would be the "recommended" method of performing these queries, considering that (1) the current modules use `existsSync` but could instead take a callback and just use `exists` (is there a strong reason for the sync?), but (2) the node docs don't like `exists` at all and state the intention to deprecate it (http://nodejs.org/api/fs.html#fs_fs_exists_path_callback).